src/main/java/com/inquirer/dao/QuestionDaoSQL.java

```diff
@@ -0,0 +1,138 @@
+package com.inquirer.dao;
+
+import com.inquirer.models.Answer;
+import com.inquirer.models.Question;
+import com.inquirer.models.User;
+import com.mysql.fabric.jdbc.FabricMySQLDriver;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+public class QuestionDaoSQL implements QuestionDao {
+
+    private static final String SELECT_ALL_USER_QUERY = "SELECT * FROM user";
+    public static final String SELECT_ALL_QUESTION_QUERY = "SELECT * FROM question";
+    public static final String SELECT_ANSWER_FOR_QUESTION_QUERY = "SELECT * FROM answer WHERE question = ?";
+    public static final String ADD_USER_QUERY = "INSERT INTO user(name,age) VALUES (?,?)";
+
+    private String username;
+    private String password;
+    private String url;
+    private Properties property;
+
+    public QuestionDaoSQL(){
+        loadProperties();
+        loadDriver();
+    }
+    private void loadProperties(){
+        property = new Properties();
+        try(InputStream propertiesFile = getClass().getClassLoader().getResourceAsStream("config.properties")) {
+            property.load(propertiesFile);
+
+            username = property.getProperty("db.user");
+            password = property.getProperty("db.password");
+            url = property.getProperty("db.url");
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+    private void loadDriver(){
+        try {
+            Driver driver = new FabricMySQLDriver();
+            DriverManager.registerDriver(driver);
+        } catch (SQLException e){
+            e.printStackTrace();
+        }
+    }
+
+    public void addUser(User user) throws SQLException {
+
+        try (Connection con = DriverManager.getConnection(url,username,password);
+            PreparedStatement statement = con.prepareStatement(ADD_USER_QUERY)) {
```

Comment:
Extract this 2 line in separate method. You repeat them every time with different queries for statement